app/Http/Controllers/PostController.php

```diff
@@ -263,6 +263,39 @@ public function postBestAnswer(Request $request)
         }
     }
 
+    public function postReport(Request $request)
+    {
+        $input = $request->all();
+
+        if ($request->ajax() && !empty($input['post_id']) && Auth::check()) {
+            $input['user_id'] = Auth::id();
+            $input['comment'] = config('constants.REPORT_MESSAGES.' . $input['report'], null);
+            $input['comment'] = $input['comment'] . '|||' .$input['note'];
+
+            $validator = \Validator::make($input,
+            [
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.Indent</br>**Severity**: error</br>**File**: app/Http/Controllers/PostController.php</br>**Line**: 276</br>Multi-line function call not indented correctly; expected 16 spaces but found 12